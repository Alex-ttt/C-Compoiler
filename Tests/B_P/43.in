void func(int *a){}

void main(){
	int a[20];
	func(a);	
}