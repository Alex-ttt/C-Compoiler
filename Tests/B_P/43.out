 Symbols Table:
   func  void function func(
          a pointer to int
  )
    {
      << block is empty >>
    }
   main  void function main(
  )
    {
      Statements:
      func
      (
        a
      )
      Symbols Table:
              a array[20] of int
    }
